conf/defaults.ini

```diff
@@ -260,6 +244,21 @@ external_manage_info =
 viewers_can_edit = false
 
 [auth]
+# Login cookie name
```

Comment:
having these settings under `[auth]` makes sense :)

Perhaps we should rename the `login` prefix?